vulkaninfo/vulkaninfo.cpp

```diff
@@ -859,7 +859,13 @@ void print_usage(const char *argv0) {
     std::cout << "--summary           Show a summary of the instance and GPU's on a system.\n\n";
 }
 
+#ifndef VK_USE_PLATFORM_IOS_MVK
```

Comment:
Nitpick, but I find it nicer to not use double negatives.
Here, there would be `#if defined(IOS_MVK)` vulkanInfoMain(...) #else main(...)`